src/libraries/Common/src/Interop/OSX/Interop.libproc.cs

```diff
@@ -378,5 +385,8 @@ internal static unsafe rusage_info_v3 proc_pid_rusage(int pid)
 
             return info;
         }
+
+        private static bool IsAppSandboxEnabled()
```

Comment:
Is this the canonical way?

Reply:
I'm not sure what exactly you're referring to. Do you mean the usage of a method vs. a property, the name of the method, both, or something else?

Reply:
I think this was about whether checking the `APP_SANDBOX_CONTAINER_ID` env var is the canonical way to tell whether you're in the sandbox - yes, that seems to be the best/easiest way to tell.

Reply:
Yes apologies for not being clear. OK.